spec.go

```diff
@@ -127,7 +127,12 @@ type Artifacts struct {
 	Binaries map[string]ArtifactConfig `yaml:"binaries,omitempty" json:"binaries,omitempty"`
 	// Manpages is the list of manpages to include in the package.
 	Manpages map[string]ArtifactConfig `yaml:"manpages,omitempty" json:"manpages,omitempty"`
+	// ConfigDirectories is a list of directories the RPM should place under %{_sysconfigdir}	.
+	ConfigDirectories []string `yaml:"configDirectories,omitempty" json:"configDirectories,omitempty"`
```

Comment:
Maybe we can do something like `createDirectories`

which is a type like so:

```go
type CreateArtifactDirectories struct {
    Config: map[string]ArtifactDirConfig // These go in /etc
    State: map[string]ArtifactDirConfig // These go in /var/lib
}
```

Reply:
@cpuguy83  what would you see modeled in map form?

```
artifacts:
     createDirectories: 
          config: 
              <what's the map look like here, how do we want to use subpath and name?>
         state:
               <what's the map look like here, how do we want to use subpath and name?>
    ```

Reply:
Map key would be the dir name to create.

Reply:
What would the other ArtifactConfig things be used for? or would we just pretty much never include them?  

```
createDirectories: 
          config: 
               kubernets: {}
               kubernetes/manifests: {}
```

or 

```
createDirectories: 
          config: 
               kubernets: 
                   subpath: manifests
```

Reply:
oh sorry, you said `ArtifactDirConfig` which could be other things my bad, sorry!

Reply:
maybe for artifact dir config we want to optionally specify mode (i.e. mkdir -m <specified>) ?

Reply:
Yep.